src/buildHTML.js

```diff
@@ -1132,30 +1132,26 @@ groupTypes.sqrt = function(group, options) {
     if (inner.height === 0 && inner.depth === 0) {
         body = makeSpan();
     } else {
+        inner.style.paddingLeft = img.surdWidth + "em";
+        // Overlay the image and the argument.
         body = buildCommon.makeVList([
-            {type: "elem", elem: inner},
-            {type: "kern", size: lineClearance},
-            {type: "elem", elem: line},
-            {type: "kern", size: ruleWidth},
```

Comment:
It hasn't gone anywhere that I know of. The proposed `sqrt` images are overlaid on the arguments. That is, the image spans vertically from above the argument to below the argument. They are not stacked above the arguments like the old `sqrt-line` span. So I called `makeVList` in individualShift mode instead of in firstBaseline mode. 

So the question is: What is the proper value of `imgShift`? I used:
```
const imgShift = img.height - inner.height - lineClearance - ruleWidth;
````
... which is my attempt to reproduce the previous behavior.